tests/evals/test_evaluator_common.py

```diff
@@ -219,7 +219,7 @@ async def test_llm_judge_evaluator(mocker: MockerFixture):
 
     # Test without input
     evaluator = LLMJudge(rubric='Content contains a greeting')
-    result = await evaluator.evaluate(ctx)
+    result = cast(Any, await evaluator.evaluate(ctx))['LLMJudge']
```

Comment:
Why are those cast needed?

Reply:
because evaluators can return lots of things and I don't want to add 100 type checks